pkg/database/realm.go

```diff
@@ -571,6 +571,38 @@ func (r *Realm) FindAuthorizedApp(db *Database, id interface{}) (*AuthorizedApp,
 	return &app, nil
 }
 
+// ListMobileApps gets all the mobile apps for the realm.
+func (r *Realm) ListMobileApps(db *Database) ([]*MobileApp, error) {
+	var apps []*MobileApp
+	if err := db.db.
+		Unscoped().
+		Model(r).
+		Order("mobile_apps.deleted_at DESC, LOWER(mobile_apps.name)").
+		Related(&apps).
+		Error; err != nil {
+		if IsNotFound(err) {
+			return nil, nil
```

Comment:
```suggestion
			return apps, nil
```

We want to give people the empty list instead of nil.

Reply:
hehe. they're _mostly_ the same, but you're right -- not exactly.